### PR TITLE
Update article_sap_trento.xml

### PR DESCRIPTION
**Pull request by dariavladykina**
### PR creator: Description

Review from proofreader.

### PR creator: Are there any relevant issues/feature requests?

n/a

### CODE REVIEW

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -891,15 +895,15 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
       </step>
       <step>
         <para>
-          Click the corresponding row to open the details view of this check. This shows you
-          an abstract and how to remedy the problem. Further links gives you the context.
+          Click the corresponding row to open the <guimenu>Details</guimenu> view of this check. This shows you
+          an abstract and how to remedy the problem. The Further links view gives you the context.
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:899]:
```suggestion
          Click the corresponding row to open the <guimenu>Details</guimenu> view of this check. This shows you
          an abstract and how to remedy the problem. The References section contains links to the documentation
          of the different vendors to provide more context when necessary.
```

**Review by abravosuse — CHANGES_REQUESTED** [review]
Thank you for your proofreading, Daria! Just a couple of change requests on your changes.

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -792,10 +791,11 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         <para>In the left panel, click <guimenu>Cluster</guimenu>.</para>
       </step>
       <step>
-        <para>In the list, search for a &hana; cluster.</para>
+        <para>In the list, search for an &hana; cluster.</para>
```
[the remark this answers is not in the document]
Comment (dariavladykina) [reply trento/xml/article_sap_trento.xml:794]: Hi @abravosuse , the entity &hana; stands for SAP HANA, please see [here](https://github.com/SUSE/doc-unversioned/blob/main/openscap/xml/generic-entities.ent). Therefore the 'an' article is appropriate here.

[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:794]: Hi @dariavladykina, ah, in that case my bad. Please ignore my request.

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -891,15 +895,16 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
       </step>
       <step>
         <para>
-          Click the corresponding row to open the details view of this check. This shows you
-          an abstract and how to remedy the problem. Further links gives you the context.
+          Click the corresponding row to open the <guimenu>Details</guimenu> view of this check. This shows you
+          an abstract and how to remedy the problem. The References section contains links to the documentation
```
Comment (dariavladykina) [inline trento/xml/article_sap_trento.xml:899]:
```suggestion
          an abstract and how to remedy the problem. The <guimenu>References</guimenu> section contains links to the documentation
```

Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:899]: Perfect!

**Review by abravosuse — APPROVED** [review]

**Review by dariavladykina — COMMENTED** [review]
One more UI element added

**Review by abravosuse — APPROVED** [review]

**Review by tomschr — APPROVED** [review]
Thank you Daria and Alberto for all your input! Much appreciated.

I'm fine with all. Next step is to merge it into main. Many thanks!

**Pull request merged**